daemon/daemon.go

```diff
@@ -1378,8 +1378,36 @@ func (d *Daemon) bootstrapFQDN(restoredEndpoints *endpointRestoreState) (err err
 			// the ToFQDN-UUID one).
 			_, err := d.PolicyAdd(generatedRules, &AddOptions{Replace: true, Generated: true})
 			return err
+		},
+		PollerResponseNotify: func(lookupTime time.Time, qname string, response *fqdn.DNSIPRecords) {
+			// FIXME: Not always true but we don't have the protocol information here
+			protocol := accesslog.TransportProtocol(u8proto.ProtoIDs["udp"])
+
+			record := logger.LogRecord{
+				LogRecord: accesslog.LogRecord{
+					Type:              accesslog.TypeResponse,
+					ObservationPoint:  accesslog.Ingress,
+					IPVersion:         accesslog.VersionIPv4,
+					TransportProtocol: protocol,
+					Timestamp:         time.Now().UTC().Format(time.RFC3339Nano),
+					NodeAddressInfo: accesslog.NodeAddressInfo{
+						IPv4: node.GetExternalIPv4().String(),
+						IPv6: node.GetIPv6().String(),
+					},
+				},
+			}
+
+			logger.LogTags.Verdict(accesslog.VerdictForwarded, "DNSPoller")(&record)
+			logger.LogTags.DNS(&accesslog.LogRecordDNS{
```

Comment:
We should add a field to `LogRecordDNS` that will indicate that this is not a DNS lookup made by the app but poller, e.g. `PollerTriggered bool`.

Reply:
I opted for something a little more general (i.e. a string) so that we can include future sources like api-update or magic-service-discovery-crd, I can just use the boolean, though, if you feel this is a little too much.